Investigate test failure: trx_finality_status_test

https://github.com/eosnetworkfoundation/mandel/runs/7326242323?check_suite_focus=true

```
  File "/home/runner/work/mandel/mandel/build/tests/trx_finality_status_test.py", line 149, in <module>
    assert state == localState, \
AssertionError: ERROR: getTransactionStatus didn't return "LOCALLY_APPLIED" state.
```
Logs are attached to the failed run.

--- a/tests/trx_finality_status_test.py
+++ b/tests/trx_finality_status_test.py
@@ -47,7 +47,6 @@ dontKill=args.leave_running
 prodCount=1
 killAll=args.clean_run
 walletPort=TestHelper.DEFAULT_WALLET_PORT
-totalNodes=pnodes+1
 
 walletMgr=WalletMgr(True, port=walletPort)
 testSuccessful=False

